testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/ApplicationsTest.java

```diff
@@ -99,24 +99,23 @@ public void toggleApplicationDetailsTest() throws Exception {
         assertFalse(applications.isEmpty());
         Map<String, ApplicationsPage.ClientRepresentation> apps = applications.stream().collect(Collectors.toMap(x -> x.getClientId(), x -> x));
         assertThat(apps.keySet(), containsInAnyOrder("always-display-client", "account-console"));
-        assertClientRep(apps.get("account-console"), "Account Console", false, true, getAuthServerRoot() + "realms/test/account/", true);
-        assertClientRep(apps.get("always-display-client"), "Always Display Client", false, false, getAuthServerRoot() + "realms/master/app/always-display-client", false);
+        assertClientRep(apps.get("account-console"), "Account Console", false, true, true);
+        assertClientRep(apps.get("always-display-client"), "Always Display Client", false, false, false);
 
         applicationsPage.toggleApplicationDetails("account-console");
         applications = applicationsPage.getApplications();
         assertFalse(applications.isEmpty());
         apps = applications.stream().collect(Collectors.toMap(x -> x.getClientId(), x -> x));
         assertThat(apps.keySet(), containsInAnyOrder("always-display-client", "account-console"));
-        assertClientRep(apps.get("account-console"), "Account Console", false, true, getAuthServerRoot() + "realms/test/account/", false);
-        assertClientRep(apps.get("always-display-client"), "Always Display Client", false, false, getAuthServerRoot() + "realms/master/app/always-display-client", false);
+        assertClientRep(apps.get("account-console"), "Account Console", false, true, false);
+        assertClientRep(apps.get("always-display-client"), "Always Display Client", false, false, false);
     }
 
-    private void assertClientRep(ApplicationsPage.ClientRepresentation clientRep, String name, boolean userConsentRequired, boolean inUse, String effectiveUrl, boolean applicationDetailsVisible) {
+    private void assertClientRep(ApplicationsPage.ClientRepresentation clientRep, String name, boolean userConsentRequired, boolean inUse, boolean applicationDetailsVisible) {
         assertNotNull(clientRep);
         assertEquals(name, clientRep.getClientName());
         assertEquals(userConsentRequired, clientRep.isUserConsentRequired());
         assertEquals(inUse, clientRep.isInUse());
-        assertEquals(effectiveUrl, clientRep.getEffectiveUrl());
```

Comment:
Should we assert the "normal" URL instead? This way I think we're not checking the URL at all.

Reply:
Yes, I'll look into that.